src/index.js

```diff
@@ -199,9 +199,17 @@ class HCaptcha extends React.Component {
     execute () {
       const { isApiReady, isRemoved, captchaId } = this.state;
 
-      if (!isApiReady || isRemoved) return
+      if (!isApiReady || isRemoved) return;
+
+      hcaptcha.execute(captchaId);
+    }
+
+    async executeAsync () {
+      const { isApiReady, isRemoved, captchaId } = this.state;
+      
+      if (!isApiReady || isRemoved) return;
 
-      hcaptcha.execute(captchaId)
+      return hcaptcha.execute(captchaId, { async: true });
```

Comment:
I would move this to match the current pattern of hCaptcha API by passing in the config async into the execute method above.

Reply:
Yep, should simplify this all together.